lib/v2/flyert/utils.js

```diff
@@ -16,19 +16,25 @@ async function load(link) {
     $('script, style').remove();
     $('#loginDialog').remove();
 
+    // 获取第一个帖子对象
+    const firstpost = $('.firstpost');
+
     // 修改图片中的链接
-    $('ignore_js_op img').each(function () {
-        const new_src = $('ignore_js_op img').attr('file');
-        $(this).attr('src', new_src);
+    firstpost.find('ignore_js_op img').each(function () {
+        $(this).attr('src', $(this).attr('file'));
+        // remove useless atrributes
+        for (const attr of ['id', 'aid', 'zoomfile', 'file', 'zoomfile', 'class', 'onclick', 'title', 'inpost', 'alt', 'onmouseover']) {
+            $(this).attr(attr, null);
```

Comment:
You can use  [removeAttr](https://cheerio.js.org/docs/api/classes/Cheerio#removeattr) instead of setting attributes to null.

Reply:
fixed.